### DISCUSSION TITLE
How to prohibit None value for Optional fields?

### DISCUSSION THREAD

**Discussion by mertbakir** [fastapi/fastapi#4902] [Questions] [answered]
Hi, I have probably a very ez case, but couldn't find the answer on the net. 

I have a field, that has to be integer and >=0 , and I don't want it to be mandatory. So if no value is set, it should be 0 by default.

This is what I did:
`some_field: Optional[int] = Field(default=0, gte=0)`

But when client sends: `some_field: null` in the payload it becomes `None`. Do I have to check this case using validators or in the constructor function or is there a better way handle it using fastapi?

**Comment by rafsaf** [comment, marked as answer]
`some_field: int = Field(default=0, ...)`

:P

**Reply by mertbakir** [reply]
lol, thanks. so, `optional` and `default=None` is the same.
